readme.md

```diff
@@ -149,19 +149,25 @@ renders in the deployed environment, where these work due to:
 npm add_user
 ```
 
-### Publish
+### Versions and publishing
 
-Continuous Integration will automatically publish `main` to npm when you push a `v*` tag.
+Continuous Integration will now create a new version, build and publish it to npm
```

Comment:
```suggestion
CircleCI will now create a new version, build and publish it to npm
```

Just being picky about the language. Maybe the battle is lost now but I like to reserve the term _continuous integration_ for the development style that would pass this test, not tooling:

![image](https://user-images.githubusercontent.com/159481/215780691-7c78824c-585e-42eb-9eb0-7453bd7b3b9b.png)